File: backend/MiniOrder.Domain/Entities/Brand.cs
namespace MiniOrder.Domain.Entities;

public class Brand
{
    public int Id { get; set; }

    public string Name { get; set; } = string.Empty;

    public string Slug { get; set; } = string.Empty;

    public bool IsActive { get; set; }

    public ICollection<Product> Products { get; set; } = new List<Product>();
}

File: frontend/src/types/brand.ts
export interface Brand {
  id: number;
  name: string;
  slug: string;
}

export interface BrandDetail {
  id: number;
  name: string;
  slug: string;
  isActive: boolean;
}

export interface CreateBrandRequest {
  name: string;
  slug: string;
}

export interface UpdateBrandRequest {
  name: string;
  slug: string;
  isActive: boolean;
}
